server/utils/queryHelpers/communitySanitize.ts

```diff
@@ -1,18 +1,18 @@
 export default (communityData, locationData, loginData, scopeData) => {
 	const cleanedData = { ...communityData };
-	const { canManageCommunity } = scopeData.activePermissions;
+	const { canManageCommunity, canViewCommunity } = scopeData.activePermissions;
 	const availablePages = {};
 
 	cleanedData.collections = cleanedData.collections
 		.filter((item) => {
 			/* Collection access is granted when */
 			/* 1. it is public collection, or */
-			/* 2. the user is a community manager, or */
+			/* 2. the user can view a community or */
```

Comment:
I might word this as `the user has view permissions in a community` since "can view" could also mean "able to load the community homepage".